dtfs-central-api/api-tests/v1/utilisation-reports/post-keying-data.api-test.ts

```diff
@@ -392,12 +392,12 @@ describe(`POST ${BASE_URL}`, () => {
     expect(entities[0].id).toEqual(facilityId);
     /**
      * The fixed fee is calculated as follows:
-     * fixed fee = utilisation * bank fee adjustment * interest percentage * days left in cover period / day count basis
+     * fixed fee = (utilisation * (coverPercentage / 100)) * bank fee adjustment * interest percentage * days left in cover period / day count basis
      *           = 1 * 0.9 * 100 / 100 * 730 / 1
      *           = 0.9 * 730
      *           = 657
      */
-    expect(entities[0].fixedFee).toEqual(657);
+    expect(entities[0].fixedFee).toEqual(525.6);
```

Comment:
Can we please update the calculation in the comment to include all the correct numbers e.g. whatever value the cover percentage is? It's important that future testers can redo the spelled calculation manually to see that the test assertion makes sense. 

Here if we define the cover percentage to be 100 on the tfm facility then it simplifies to 1 and we can keep the calculation as simple to follow and focused on the amendment as possible which is the purpose of the test.